srcpkgs/vorta/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'vorta'
+pkgname=vorta
+version=0.8.10
+revision=1
+build_style=python3-module
+#'test_export_fail_unwritable' fails as root.
+#'test_ssh_dialog' fails under xbps-src. 
+make_check_args="
+ --deselect=tests/test_import_export.py::test_export_fail_unwritable
+ --deselect=tests/test_repo.py::test_ssh_dialog"
+hostmakedepends="python3-setuptools python3-wheel"
+depends="borg python3-peewee python3-appdirs python3-paramiko
+ python3-SecretStorage python3-psutil python3-PyQt5"
+checkdepends="${depends} dbus python3-pytest python3-pytest-qt
+ python3-pytest-mock python3-six"
```

Comment:
why is six required here?

Reply:
Checking fails complaining the missing module `six`. My guess is that older versions of `paramiko` depended on `six`which is no longer true.  These are the latest error lines :

```
rom paramiko.pkey import PKey
/usr/lib/python3.11/site-packages/paramiko/pkey.py:30: in <module>
    import six
E   ModuleNotFoundError: No module named 'six'
```

Reply:
Then six should be added to paramiko's dependencies, this is clearly not a dependency of vorta.

Reply:
I suspect this is the offending PR: https://github.com/void-linux/void-packages/commit/136c139205cc564b00441f00a2734d2533f54469

What should I do? Open a PR to include six back?

Reply:
That commit is correct, but it removed six from paramiko's dependency tree. It should have always been an explicit dependency. Could you open a PR to add six to paramiko's dependencies?

Reply:
Just opened a PR: https://github.com/void-linux/void-packages/pull/42085.